internal/server/controller.go

```diff
@@ -1,7 +1,8 @@
-package controller
+package server
```

Comment:
IMO `server`  should contain the routers but not controllers
can have a sub-package to put all controller there?

Reply:
So I was wondering if u could so a sub-package in go.
Cuz that's what I would have done in Java

Reply:
yes but it is not a sub-package
it is treated as a new package lol

Reply:
Yeah that's why packages sucks in go, they are generic and it enforces the use of huge files (IMO)